ddtrace/_trace/processor/__init__.py

```diff
@@ -135,30 +135,55 @@ def unregister(self):
 
 @attr.s
 class TraceSamplingProcessor(TraceProcessor):
-    """Processor that keeps traces that have sampled spans. If all spans
-    are unsampled then ``None`` is returned.
+    """Processor that runs both trace and span sampling rules.
```

Comment:
This change was necesarry to have trace sampling run before span sampling, I like having all of the sampling run in one processor, but if people have other opinions, happy to discuss.